.github/workflows/tests.yml

```diff
@@ -30,8 +30,10 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: '3.7'
+      - name: Install poetry
+        run: pip install poetry
       - name: Install black
-        run: pip install $(grep black requirements-dev.txt)
+        run: pip install black==$(poetry show | grep '^black\s' | grep -Po '\d+\.\d+\.\d+')
```

Comment:
This is so we don't have to install all the deps in this task by doing `poetry run black`, right? (And same with flake8 below)

Reply:
Yes, I kept that performance-trick. I was torn between just doing `poetry run black` and keeping that performance trick. Should I switch to `poetry run`?